assets/docs/vasudevgupta7/models/wav2vec2-robust/1.md

```diff
@@ -0,0 +1,50 @@
+# Module vasudevgupta7/wav2vec2-robust/1
+
+Pre-trained speech model (without any head) from Facebook for Automatic Speech Recognition
+
+<!-- asset-path: https://storage.googleapis.com/gsoc-weights/wav2vec2_robust.tar.gz -->
+<!-- task: audio-stt -->
+<!-- network-architecture: wav2vec2 -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: true -->
+<!-- license: apache-2.0 -->
+<!-- language: en -->
+
+## Overview
+
+This model is TensorFlow equivalent of PyTorch [`facebook/wav2vec2-large-robust`](https://huggingface.co/facebook/wav2vec2-large-robust). It was published in [1].
+
+**How to use this model?**
+
+Add randomly initalized LM head over the top of pre-trained model & fine-tune the whole model.
+
+```python
+import tensorflow as tf
+import tensorflow_hub as hub
+
+# For using this pre-trained model for training, pass `trainable=True` in `hub.KerasLayer`
+pretrained_layer = hub.KerasLayer("https://tfhub.dev/vasudevgupta7/wav2vec2-robust/1", trainable=True)
+
+VOCAB_SIZE = 32
+
+# Let's wrap all the layers into `tf.keras.Model` using TensorFlow's Functional API
+speech = tf.keras.Input(shape=(246000,))
```

Comment:
Can you define these magic numbers as constants and explain them this way? For example
```python
WAV_DATA_POINTS = 246000
```